submissions/dashakim/a-tiny-world2/index.js

```diff
@@ -4,13 +4,19 @@ class Inhabitant {
     this.species = props.species;
     this.gender = props.gender;
     this.saying = props.saying;
-    // this.friends = []
+    this.friends = [];
+  }
+  setFriends(friends) {
+    this.friends = friends;
```

Comment:
Dirty trick to make your function accepting both Array and any number of arguments wrapping them into an Array:
```
function doSmth(...params) {
  params = params.flat();
  return params;
}

doSmth([1,2,3]); // [1,2,3]
doSmth(1,2,3); // [1,2,3]
doSmth(1); // [1]
```